Library/Formula/stack.rb

```diff
@@ -0,0 +1,15 @@
+class Stack < Formula
+  desc "A cross-platform program for developing Haskell projects."
+  homepage "https://github.com/commercialhaskell/stack"
+  url "https://github.com/fpco/stack/releases/download/v0.0.0-beta/stack-0.0.0-x86_64-osx.gz"
```

Comment:
One of the reasons for this tool is to simplify the building of Haskell applications, by bypassing the dependency hell of package and compiler versions. As the tool itself is a haskell tool, those issues come with compiling the tool.

GHC is a large download, and with all of the dependencies of this project, compiling from source would require very large downloads, a long compile time, and unless this was all done in a sandbox environment and thrown away at the end, it would also leave a large number of possibly conflicting packages lying around the system.

It appears that the developers of Stack are encouraging the use of the binaries, and it is for these reasons that I chose to use the binaries.